### ISSUE TITLE
Add grassland landuse polygon near Martinez, California (and other landuse)

### ISSUE THREAD

**Issue by nvkelso** [tilezen/vector-datasource#1259]
tl;dr

Add just the critical missing features in this issue for landcover "green" areas listed here:

- [ ] natural=`grassland` into **landuse** layer as polygon
- [ ] natural=`sand` into **landuse** layer as polygon
- [ ] natural=`shingle` into **landuse** layer as polygon
- [ ] natural=`heath` into **landuse** layer as polygon
- [ ] natural=`bare_rock` into **landuse** layer as polygon
- [ ] natural=`desert` into **landuse** layer as polygon
- [ ] landuse=`vineyard` into **landuse** layer as polygon

And all the other "nice to have" changes in https://github.com/tilezen/vector-datasource/issues/1259#issuecomment-450447362 should be moved to v1.8 milestone.

===

* **What did you see?**

Missing grassland **landuse** layer polygons near Martinez, California

* **What did you expect to see?**

`natural=grassland` polygons. Looks like we're missing that from our filters.

* **What map location are you having problems with?** 

- http://www.openstreetmap.org/relation/6947071#map=16/37.9948/-122.0820

* **Screenshot?**

<img width="563" alt="screen shot 2017-05-25 at 19 03 56" src="https://cloud.githubusercontent.com/assets/853051/26474081/f40e8eae-417c-11e7-85e5-6ce52de2ae2a.png">

<img width="806" alt="screen shot 2017-05-25 at 19 03 43" src="https://cloud.githubusercontent.com/assets/853051/26474082/f4120b2e-417c-11e7-8ab5-7c8d991a895e.png">

**Comment by nvkelso** [comment]
Also do a quick scan of other `natural=*` values we should add:

- https://taginfo.openstreetmap.org/keys/natural#values

**Comment by nvkelso** [comment]
Related to https://github.com/tilezen/vector-datasource/issues/1253 for adding `wetland` tag.

**Comment by nvkelso** [comment]
For reference, OSM Carto has low zoom and high zoom landuse definitions:

- https://github.com/gravitystorm/openstreetmap-carto/blob/3cbd1112b9d82f361ca95e660e64b2b63f1cfce2/project.mml#L82-L165

**Comment by zerebubuth** [comment]
:+1: looks good to me. A few of those (`animal_keeping`, `livestock`) only have a few thousand usages, which makes me wonder how stable they are. But we can figure that out later.

**Comment by nvkelso** [comment]
Filed: https://github.com/tilezen/vector-datasource/issues/1781 for followup for random OSM Carto landcovers.

**Comment by nvkelso** [comment]
QA verified.
